### PR TITLE
[DOC-139] Docker Page Migration

### PR DESCRIPTION
**Pull request by chrisgocode**

### CODE REVIEW

File: docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+for more information.
+
+Includes the following web applications:
+
+* Ed-Fi Web API
+* Ed-FI ODS Admin Api
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![SingleTenant Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-PostgreSql.png)
```
Comment (chrisgocode) [inline docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx:99]: All images in this section were "unknown" draw.io images. I did my best to match each image.

File: docs/reference/7-docker/readme.mdx [hunk 2]
```diff
@@ -0,0 +1,806 @@
...
+options are:
+
+* **odsContextRouteTemplate** - Defines the route parameter template used to
+    specify the school year.
+
+    {/* Configure it following [route
+    constraints](<https://techdocs.ed-fi.org/docker> run --rm -v
+    c:%5CEd-Fi-ODS-Docker%5CCompose-Generator%5CAlpine%5Ctemplates%5Cpgsql%5Cparameters-year-specific.yml:/parameters
+    -v C:%5CEd-Fi-ODS-Docker%5CCompose%5Cpgsql%5CSingleTenant-OdsContext:/output
+    edfialliance/ods-compose-generator). */}
```
Comment (chrisgocode) [inline docs/reference/7-docker/readme.mdx:425]: [Lines 421-425]

I'm not exactly sure what this link is supposed to do. Un-commented, it causes a build error.
There is another link like this further down.

File: docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx (hunk 1 above)
Comment (stephenfuqua) [reply docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx:99]: I think you got them all. @vimayya please review the output and confirm that the docker compose architecture drawings match up to the sections correctly.

File: docs/reference/7-docker/readme.mdx (hunk 2 above)
Comment (stephenfuqua) [reply docs/reference/7-docker/readme.mdx]: Someone had accidentally pasted a powershell command as a link. I think I found the intended URL.

File: docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+</Tabs>
+
+
+### SingleTenant Configuration
+
+This configuration shows a basic example of a single tenant configuration where
+a single ODS database serves all data going through the API. While this example
+has one ODS for demonstration purpose, single tenant deployments can support
+separate ODS per school years and districts. See [API Client and ODS Instance
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559631/API+Client+and+ODS+Instance+Configuration)
```
Comment (vimayya) [inline docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx:89]: update this link to /reference/ods-api/platform-dev-guide/configuration/api-client-and-ods-instance-configuration

File: docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-Mssql.png)
+  </TabItem>
+</Tabs>
+
+### SingleTenant with ODS Context Configuration
+
+This configuration shows a basic example of _explicit_ data segmentation
+strategy based on school year, i.e. school year becomes a required part of the
+API route segments. See [Context-Based Routing for Year-Specific
+ODS](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559277/Context-Based+Routing+for+Year-Specific+ODS)
```
Comment (vimayya) [inline docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx:115]: update this link to reference/ods-api/platform-dev-guide/configuration/context-based-routing-for-year-specific-ods

File: docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenantContext-PostgreSql.png)
+  </TabItem>
+</Tabs>
+
+### MultiTenant Configuration
+
+This configuration shows a basic example of multiple tenant configuration, with
+one Admin, Security, ODS databases per tenant. See [Multi-Tenant
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559218/Single+and+Multi-Tenant+Configuration)
```
Comment (vimayya) [inline docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx:135]: Update this link to reference/ods-api/platform-dev-guide/configuration/single-and-multi-tenant-configuration

File: docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+</Tabs>
+
+### MultiTenant with ODS Context Configuration
+
+This configuration shows a basic example of multi tenant configuration with
+_explicit_ data segmentation strategy based on school year. Deployment has one
+Admin, Security per tenant and has one ODS per school year. ODS for the school
+year is selected based on the schoolyear in the API route segment. See
+[Multi-Tenant
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559218/Single+and+Multi-Tenant+Configuration)
```
Comment (vimayya) [inline docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx:157]: Update this link to reference/ods-api/platform-dev-guide/configuration/single-and-multi-tenant-configuration

File: docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+and [Context-Based Routing for Year-Specific
+ODS](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559277/Context-Based+Routing+for+Year-Specific+ODS)
```
Comment (vimayya) [inline docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx:159]: update this link to reference/ods-api/platform-dev-guide/configuration/context-based-routing-for-year-specific-ods

**Review by roberthunterjr — COMMENTED** [review]
Looks good. Ready to approve after link updates are completed.

**Comment by stephenfuqua** [comment]
> Are we not migrating older https://edfi.atlassian.net/wiki/spaces/EDFITOOLS/pages/24119348/Docker+Deployment+2.x pages?

Somehow that information was missing from the export, which is why Chris did not include it. I'm taking a look.

**Review by vimayya — APPROVED** [review]

**Pull request merged**